samples/dashboards/demo/minimal/demo.js

```diff
@@ -21,18 +21,45 @@ Dashboards.board('container', {
         layouts: [{
             rows: [{
                 cells: [{
-                    id: 'title'
-                }]
-            }, {
-                cells: [{
+                    responsive: {
```

Comment:
Is the data grid a bit tall for the content?

<img width="542" alt="Screen Shot 2023-08-19 at 15 19 43" src="https://github.com/highcharts/highcharts/assets/5276642/296f8ce6-7305-46dc-a08f-ec12d4a7d112">

Reply:
that's something that happens to the DataGridComponent by default. Something that we might want to add to backlog.